### PR TITLE
New package: rustup-1.9.0

### PR DESCRIPTION
**Pull request by dleeram**
Outdates #5943.
In this approach, the user mantains the toolchains.

### CODE REVIEW

**Comment by Vaelatern** [comment]
Is the `INSTALL.msg` superfluous when you consider other upstream docs?

**Comment by dleeram** [comment]
I agree that upstream documentation is better. Maybe a wiki page explaining the different methods of using Rust on Void.

File: srcpkgs/rustup/template [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Template file for 'rustup'
+pkgname=rustup
+version=1.9.0
+revision=1
+wrksrc="${pkgname}.rs-${version}"
+conflicts="cargo rust"
```
Comment (Vaelatern) [inline srcpkgs/rustup/template]: Can you explain the `conflicts`?

Comment (Vaelatern) [reply srcpkgs/rustup/template]: (Travis didn't build just now so I can't just look at the files there)

Comment (dleeram) [reply srcpkgs/rustup/template]: The rustup binary is a [proxy](https://github.com/rust-lang-nursery/rustup.rs/blob/38e1d3a6e5049b510f371c7c55972b35eb8a3132/src/rustup-cli/main.rs#L3-L9) depending on arg0 when we run it. We provide `/usr/bin/cargo` and `/usr/bin/rustc` as symlinks to the rustup binary to use this functionality.

**Pull request merged**